.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,6 +11,11 @@
 
 ## Docs
 
-<!-- Did you make a user-facing change? You probably need to update docs! -->
-<!-- Add a link to your docs PR here. If no docs added, explain why (e.g. "bug fix only") -->
-<!-- Link: https://github.com/withastro/docs -->
+<!-- Is this a visible change? You probably need to update docs! -->
+<!-- DON'T DELETE THIS SECTION! If no docs added, explain why. -->
+<!-- https://github.com/withastro/docs -->
+
+## Changeset
+
+<!-- Don't forget to add a changeset! -->
+<!-- Run: pnpm exec changeset -->
```

Comment:
Maybe add these reminder comments to the top of the template? Comments won't be printed but the `## Changeset` heading will, which might just be noisy.

Reply:
moved to the top, wdyt?